Gemfile

```diff
@@ -60,9 +60,10 @@ gem 'font-awesome-rails'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
-
+  gem "rspec-rails", :group => [:development, :test]
```

Comment:
This `gem` is already in the `:development` and `:test` group. There is no need to specify it twice. Also, other gems use `'`. Don't use `"`